app/views/mixtapes/edit.html.erb

```diff
@@ -52,6 +52,7 @@
 
 <div id='current-upload'>
   <p>Drag one or more audio files here to add it to the mixtape. Drag them around with the handles to reorder them.</p>
+  <input type="file" id="input">
```

Comment:
Shoot, left that in by accident. I was having a hard time dragging files w/o a file manager (linux).